packages/themes/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "travix-themes",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Travix International",
+  "license": "MIT"
```

Comment:
this should be `CC-BY-SA` licence from now on.. based on: https://en.wikipedia.org/wiki/Creative_Commons_license#Seven_regularly_used_licenses

Reply:
done